README.md

```diff
@@ -64,12 +65,55 @@ using json = nlohmann::json;
 
 to the files you want to use JSON objects. That's it. Do not forget to set the necessary switches to enable C++11 (e.g., `-std=c++11` for GCC and Clang).
 
+### CMake
+
+If you are using CMake, just download the latest release, run 
+```sh
+cd <path/build/folder> 
+cmake <path/to/nlohman_json/> 
+make install 
+```
+
+To use it in your own projects ``` CMakeLists.txt ```
+
+```cmake
+find_package(nlohmann_json CONFIG REQUIRED)
+target_link_libraries(<YourTargetName> nlohmann_json)
+```
+
+### CMake with hunter (package manager)
+
+You can also use the CMake package manager [hunter](https://github.com/ruslo/hunter) to automatically download ``` nlohman_json ``` for you.
+```cmake
+HunterGate( # must be before project()
+    URL "https://github.com/ruslo/hunter/archive/v0.19.52.tar.gz"
+    SHA1 "6d3a8135ba62726d810fd8c8c7d97980fa6c3b4a"
+)
+[...]
+hunter_add_package(nlohmann_json)
+find_package(nlohmann_json CONFIG REQUIRED)
+target_link_libraries(... nlohmann_json)
```

Comment:
This is quite verbose. Is it possible to describe this shorter as it is done for Meson, Conan, or Homebrew?